apm/src/app/user/user.component.html

```diff
@@ -0,0 +1,18 @@
+<h1>Edit Profile</h1>
+<hr>
+<div class="col-md-4">
+  <form autocomplete="off" [formGroup]="editForm" (ngSubmit)="UpdateUserInfo()" novalidate>
+    <div class="form-group">
+      <label for="firstName">First Name:</label>
+      <input formControlName ="firstName" name="firstName" id="firstName" type="text" class="form-control" placeholder="First Name..." />
+      <em *ngIf="!FirstName">Required</em>
```

Comment:
I believe this is for showing the validation, so the naming could be something like isFirstNameValid or isFirstNameBlank